cards.json

```diff
@@ -3,6 +3,10 @@
         "image": "assets/Images/accordion.png",
         "link": "Public/accordion.html"
     },
+ {
```

Comment:
it's not properly added, also follow alphabetical order.

Reply:
Have made the changes. Please review it.